src/main.cpp

```
@@ -2,12 +2,11 @@
 #include "main_window.h"
 #include "zigbee_provider.h"
 
-#include <iostream>
 
 #include <QApplication>
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)